surreal.js

```diff
@@ -36,37 +36,65 @@ var $ = { // You can use a different name than "$", but you must change the refe
 		e.attributes    = e.attribute
 		e.attr          = e.attribute
 
+		// Tree Traversal.
+		e.el = (selector, start = e, warning = true) => $.el(selector, start, warning)
+		e.any = (selector, start = e, warning = true) => $.any(selector, start, warning)
+		Object.defineProperty(e, 'parent', {
+			get: () => $.sugar(e.parentElement),
+			configurable: true
+		})
+
+
 		// Add all plugin sugar.
 		$._e = e // Plugin access to "e" for chaining.
-		for (const [key, value] of Object.entries(sugars)) {
+		for (const [key, value] of Object.entries($.sugars)) {
 			e[key] = value.bind($) //e[key] = value
 		}
 
 		return e
 	},
 
-	// Return single element. Selector not needed if used with inline <script> 🔥
-	// If your query returns a collection, it will return the first element.
+	// Return current element. Works inside <script> or event handlers created with surreal 🔥
 	// Example
 	//	<div>
 	//		Hello World!
-	//		<script>me().style.color = 'red'</script>
+	//		<script>me.style.color = 'red'</script>
 	//	</div>
-	me(selector=null, start=document, warning=true) {
-		if (selector == null) return $.sugar(start.currentScript.parentElement) // Just local me() in <script>
-		if (selector instanceof Event) return $.me(selector.target) // Events return event.target
+	get me() {
+		if (document.currentScript) return $.sugar(document.currentScript.parentElement) // Just local me in <script>
+		else return $._evtEl // If we are in an event handler this will be the element declaring the handler
+	},
+
+	// Return single element.
+	// If your query returns a collection, it will return the first element.
+	el(selector, start=document, warning=true) {
+		if ($.isNodeList(start)) {
+			for (const node of start) {
+				const res = $.el(selector, node, warning)
+				if (res) return res
```

Comment:
Not sure if this is the best way to chain `el` on top of `any`. Right now it just returns the first match, to keep things in line such that even when chained, `el` only returns one item.